src/open_inwoner/scss/views/App.scss

```diff
@@ -254,63 +254,56 @@
 }
 
 @font-face {
-  font-family: 'TheMix C5';
-  src: local('TheMix C5'),
-    url('/static/fonts/TheMixC5/DesktopFonts/TheMixC5-5_Plain.otf')
-      format('opentype');
+  font-family: 'Heading';
+  font-weight: bold;
+  font-style: normal;
+  src: url('/static/fonts/ubuntu/Ubuntu-B.ttf') format('truetype');
 }
 
 @font-face {
-  font-family: 'TheMix C5';
+  font-family: 'Heading';
   font-weight: bold;
-  src: local('TheMix C5'),
-    url('/static/fonts/TheMixC5/DesktopFonts/TheMixC5-7_Bold.otf')
-      format('opentype');
+  font-style: italic;
+  src: url('/static/fonts/ubuntu/Ubuntu-BI.ttf') format('truetype');
 }
 
 @font-face {
-  font-family: 'TheSans C5';
-  src: local('TheSans C5'),
-    url('/static/fonts/TheSansC5/DesktopFonts/TheSansC5-5_Plain.otf')
-      format('opentype');
+  font-family: 'Body';
+  src: url('/static/fonts/lato/Lato-Regular.ttf') format('truetype');
 }
 
 @font-face {
-  font-family: 'TheSans C5';
+  font-family: 'Body';
   font-weight: bold;
-  src: local('TheSans C5'),
-    url('/static/fonts/TheSansC5/DesktopFonts/TheSansC5-7_Bold.otf')
-      format('opentype');
+  src: url('/static/fonts/lato/Lato-Bold.ttf') format('truetype');
```

Comment:
Why do you specify `font-style: normal;` for the `Heading`, but not for the `Body`?